tests/extension/decimal/array.py

```diff
@@ -28,7 +28,7 @@
 )
 
 from pandas.core.dtypes.base import ExtensionDtype
-from pandas.core.dtypes.common import (
+from pandas.core.dtypes.common import (  # type: ignore[attr-defined]
```

Comment:
UGH.  Now that I see that this is used in a test (this one is copied from the pandas source), I think we need to open an issue in `pandas` about documenting this.  I'll do that, and keep this PR on hold.

Reply:
Created https://github.com/pandas-dev/pandas/issues/60905